bloom/filter.go

```diff
@@ -47,9 +45,9 @@ func MakeSplitBlockFilter(data []byte) SplitBlockFilter {
 //	f := make(bloom.SplitBlockFilter, bloom.NumSplitBlocksOf(n, 10))
 //
 func NumSplitBlocksOf(numValues int64, bitsPerValue uint) int {
-	numBytes := bits.ByteCount(uint(numValues) * bitsPerValue)
+	numBytes := ((uint(numValues) * bitsPerValue) + 7) / 8
```

Comment:
This might be a part of the change that is more open to discussion, the `bits.ByteCount` function was used in more than one place.

I decided to inline it because the operation was really trivial and I could not come up with a good name for a package that would host this function.

Reply:
Fine by me